Godot_test/ClientScripts/GameLevel.cs

```diff
@@ -45,13 +46,16 @@ public void CreateEntities(Dungeons.Tiles.Tile[,] tiles)
 		if (tile is Roguelike.Tiles.Interactive.Barrel)
 		{
 		  AddChildFromScene(tile, "res://Entities/Barrel.tscn");
-
+		}
+		else if (tile is Roguelike.Tiles.Interactive.Chest)
+		{
+		  AddChildFromScene(tile, "res://Entities/Chest.tscn");
```

Comment:
Roguelike.Tiles.Interactive.Chest has a member  :
 public enum ChestKind { Unset, Plain, Gold, GoldDeluxe }
We shall create a diffrenet image depend on kind.